SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -343,6 +343,27 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Runtime", AdvancedDisplay)
 	FRuntimeVariantVersion CompatibilityModeRuntimeVersion;
 
+	/** Whether to use the GDK-associated SpatialOS inspector version for local deployments, or to use the one specified in the
+	 * InspectorVersion field. */
+	UPROPERTY(EditAnywhere, config, Category = "Inspector", meta = (DisplayName = "Use GDK Pinned Inspector Version"))
+	bool bUseGDKPinnedInspectorVersion = true;
+
+	/** Runtime version to use for local deployments, if not using the GDK pinned version. */
+	UPROPERTY(EditAnywhere, config, Category = "Inspector", meta = (EditCondition = "!bUseGDKPinnedInspectorVersion"))
+	FString InspectorVersionOverride;
+
+	/** Returns the version information for the currently set inspector*/
+	const FString GetInspectorVersion() const
+	{
+		if (bUseGDKPinnedInspectorVersion) {
```

Comment:
```suggestion
		return bUseGDKPinnedInspectorVersion ? SpatialGDKServicesConstants::SpatialOSInspectorPinnedVersion : InspectorVersionOverride;
```
And then remove the rest. This is the fancy way I mentioned.